doc/api_ref/pubkey.rst

```diff
@@ -1213,10 +1403,10 @@ encapsulated key and returns the shared secret.
 Botan implements the following KEM schemes:
 
 1. RSA
-#. ML-KEM (Kyber)
+#. ML-KEM (aka Kyber)
```

Comment:
```suggestion
#. ML-KEM (formerly known as Kyber)
```

... not sure how long until a typical user wouldn't know what Kyber stands for. Or can anyone type out "Rijndael" without googling "AES| to check for the correct spelling? 🤡